test/unit/lib/modules/groupStore.test.js

```diff
@@ -480,6 +507,28 @@ describe('groupStore', () => {
       });
     });
 
+    it('should create a new  group and override the API Version', (done) => {
+      const API_VERSION = 5;
+
+      nock('https://baas.kinvey.com')
+        .matchHeader('content-type', 'application/json')
+        .matchHeader('x-kinvey-api-version', `${API_VERSION}`)
+        .post(`/group/${environmentId}/`, {
+          username: 'abc'
+        })
+        .basicAuth({
+          user: environmentId,
+          pass: mastersecret
+        })
+        .reply(200, { _id: 1234, username: 'abc' });
+
+      this.store({ apiVersion: 5 }).create({ username: 'abc' }, (err, result) => {
```

Comment:
basically all of the other individual test cases set `apiVersion` field to the value of `API_VERSION`. In this test (and a few others), you're explicitly setting it to `5`. Is this intentional?